app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,118 @@
+<% infos = [
+    { title: "preparation time (hrs)", value: @recipe.preparation_time },
+    { title: "cooking time (hrs)", value: @recipe.cooking_time }
+  ] %>
+
+<section class="p-recipe-details">
+
+  <article class="c-recipe-view c-card">
+    <header class="c-recipe-view__header">
+      <div class="c-recipe-view__author-box">
+        <img 
+          src=<%= image_url('blank_avatar.jpg') %>
+          class="c-recipe-view__author-img" 
+          alt="recipe author photo" 
+        />
+        <div class="c-recipe-view__author-info">
+          <span class="c-recipe-view__author-name">
+            <%= @recipe.user.name %>
+          </span>
+          <span class="c-recipe-view__date">
+            created <%= time_ago_in_words(@recipe.created_at) %> ago
+          </span>
+        </div>
+      </div>
+
+      <div class="c-recipe-view__action">
+        <%= link_to 'javascript:history.back()', class: "c-icon-btn" do %>
+          <svg class="c-icon--sm">
+            <use xlink:href=<%= image_url("icons/sprite.svg#chevron-left")%>></use>
+          </svg>
+        <% end %>
+        <%= render "components/delete_recipe_btn", recipe: @recipe %>
+      </div>
+    </header>
+
+    <%= render "components/attached_img", 
+      image: @recipe.image, 
+      image_class: "c-recipe-view__img",
+      image_alt: "recipe photo",
+      placeholder: "recipe_placeholder.jpg" 
+    %>
+
+    <h1 class="c-recipe-view__title c-section-title">
+      <%= @recipe.name %>
+    </h1>
+
+    <p class="c-recipe-view__description">
+      <%= @recipe.description %>
+    </p>
+
```

Comment:
[REQUIRED] I feel that your description is the same thing as the steps in the wireframe. if so then it should come after preparation and cooking time (which is in your `components/diplay_info`). If not, remove it and ensure to include steps under the preparation and cooking time just like the wireframe
![image](https://github.com/IvonneBenitesRodriguez/recipe_app/assets/62938735/f5ab26de-0d2f-4b2f-8afd-603d43573e22)